src/app/shared/sync.service.ts

```diff
@@ -23,76 +22,85 @@ const passwordFormFields = [
 @Injectable()
 export class SyncService {
 
+  private parentDomain: string;
+
   constructor(
     private couchService: CouchService,
     private userService: UserService,
-    private dialogsFormService: DialogsFormService,
-    private planetMessageService: PlanetMessageService
+    private dialogsFormService: DialogsFormService
   ) {}
 
-  private defaultConfig = {
-    'create_target':  false,
-    'continuous': false,
-    'owner': this.userService.get().name
-  };
+  confirmPasswordAndRunReplicators(replicators) {
```

Comment:
there will be cases when we do not need to ask for password confirmation. Such as when we are doing setup of application. So we will require replicator to be able to run without password confirmation